# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../coda/gcc44.patch.coda
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./coda-src/venus/vol_cml.cc.orig
+++ ./coda-src/venus/vol_cml.cc
@@ -3606,7 +3606,8 @@ static int WriteLinks(struct DirEntry *de, void *arg)
 {
     VnodeId vnode;
     Unique_t vunique;
-    char *name = de->name, *comp;
+    char *name = de->name;
+    const char *comp;
     size_t prefixlen;
     char namebuf[CODA_MAXPATHLEN];
     struct WriteLinksHook *hook = (struct WriteLinksHook *)arg;
